pom.xml

```diff
@@ -60,100 +59,91 @@
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 		<java.version>1.8</java.version>
-		
+
 		<sonar.projectKey>Batch-738-Mathew-Oberlies_RideshareBackend</sonar.projectKey>
-  		<sonar.organization>batch-738-mathew-oberlies</sonar.organization>
-  		<sonar.host.url>https://sonarcloud.io</sonar.host.url>
-		<sonar.coverage.exclusion>com/revature/models/*.java</sonar.coverage.exclusion>
-		<!-- login credentials moved to Jenkins build -->
+		<sonar.organization>batch-738-mathew-oberlies</sonar.organization>
+		<sonar.host.url>https://sonarcloud.io</sonar.host.url>
+    <sonar.coverage.exclusion>com/revature/models/*.java</sonar.coverage.exclusion>
```

Comment:
You spent time fixing alignment in this file, and this one line was changed to be misaligned.